src/Microsoft.ML.Data/Evaluators/MultiOutputRegressionEvaluator.cs

```diff
@@ -548,7 +550,7 @@ private protected override Delegate[] CreateGettersCore(DataViewRow input, Func<
             var t = schema[LabelIndex].Type as VectorType;
             if (t == null || !t.IsKnownSize || (t.ItemType != NumberDataViewType.Single && t.ItemType != NumberDataViewType.Double))
                 throw Host.ExceptSchemaMismatch(nameof(schema), "label", LabelCol, "known-size vector of float or double", t.ToString());
-            labelType = new VectorType((PrimitiveDataViewType)t.ItemType, t.Size);
```

Comment:
Just in case anyone is wondering this is in fact correct and desirable, I believe we changed things in such a way that this no longer requires a cast (since the item type if we know it is a vector type will return the primitive type).